extensions/interactions/ItemSelectionInput/directives/item-selection-input-interaction.directive.html

```diff
@@ -1,6 +1,6 @@
 <div class="item-selection-input-form-container">
   <form ng-submit="$ctrl.submitAnswer()">
-    <small ng-if="$ctrl.displayCheckboxes">
+    <small>
```

Comment:
Why was this removed?

Reply:
Please see [this](https://github.com/oppia/oppia/pull/8365#issuecomment-571983361).